private/buf/buflsp/symbol_test.go

```diff
@@ -0,0 +1,58 @@
+package buflsp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_formatComment(t *testing.T) {
```

Comment:
Don't use underscores in test names. Just name it TestFormatComment or equivalent.